src/ContainerDetails.jsx

```diff
@@ -43,6 +44,17 @@ const ContainerDetails = ({ container }) => {
                         <DescriptionListDescription>{utils.quote_cmdline(container.Config.Cmd)}</DescriptionListDescription>
                     </DescriptionListGroup>
                     }
+                    {Boolean(container.Config?.Labels?.PODMAN_SYSTEMD_UNIT) && (container.uid === 0 || container.uid === null) &&
+                    <DescriptionListGroup>
+                        <DescriptionListTerm>{_("Systemd service")}</DescriptionListTerm>
+                        <DescriptionListDescription>
+                            <Button variant="link" isInline onClick={
+                                () => cockpit.jump(`/system/services#/${container.Config?.Labels?.PODMAN_SYSTEMD_UNIT}` + (container.uid === null ? "?owner=user" : ""))}>
+                                {cockpit.format("View $0", container.Config?.Labels?.PODMAN_SYSTEMD_UNIT)}
```

Comment:
String needs i18n. Although I'm actually in favor of just dropping the "View " and showing the unit name only.  There is already a label, and "View" doesn't quite capture it anyway -- you can interact/change the unit as well, and if this needs a verb at all, it sounds more like "Go to.."?

Reply:
I'm not against considering something else, but PF suggests "View" for this @ https://www.patternfly.org/ux-writing/terminology/

> View (v.)
> 	
> Use to describe accessing another page to see details.

Reply:
Sure, no strong opinion. Let's just i18n the string.